trade-finance-manager-api/src/types/tfm-session-user.ts

```diff
@@ -1,5 +1,5 @@
-import { TfmUser } from './db-models/tfm-users';
+import { TfmUser } from '@ukef/dtfs2-common';
 
-export type TfmSessionUser = Pick<TfmUser, 'username' | 'email' | 'teams' | 'timezone' | 'firstName' | 'lastName' | 'status' | 'lastLogin'> & {
+export type TfmSessionUser = Pick<TfmUser, 'username' | 'email' | 'teams' | 'timezone' | 'firstName' | 'lastName' | 'lastLogin'> & {
   _id: string;
```

Comment:
This has had status removed. It needs to me removed from everwhere we use TfmSessionUser.